pkg/cmd/roachtest/mixed_version_jobs.go

```diff
@@ -332,7 +332,6 @@ func registerJobsMixedVersions(r *testRegistry) {
 		// is to to test the state transitions of jobs from paused to resumed and
 		// vice versa in order to detect regressions in the work done for 20.1.
 		MinVersion: "v20.1.0",
```

Comment:
does it make sense to bump the MinVersion?

Reply:
I'm... not sure if this was doing anything since old version roachtests now run on their branch.

Reply:
Ah, it was removed in https://github.com/cockroachdb/cockroach/commit/e26d040b38b3f49f6dfd9cfeef549e7529f01df5 🙌